### PR TITLE
Rephrasing for smoother reading/fix typos/wording

### PR DESCRIPTION
**Pull request by dragon788**
I was having trouble reading through the README after somebody showed me this awesome project and it seems like having it easy to read will encourage people to continue reading and try it out.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,16 +4,16 @@
 
 ## Introduction
 
-Setting up Jenkins is a complex process, as both Jenkins and it's plugin require some tunning and configuration, 
-with dozen parameters to set within the web UI `manage` section. 
+Setting up Jenkins is a complex process, as both Jenkins and it's plugin require some cunning and configuration,
```
Comment (daniel-beck) [inline README.md]: Should probably be *tuning*

Also, it's *its*, and *plugins*, plural.

File: README.md [hunk 2]
```diff
@@ -4,16 +4,16 @@
...
-Experimented Jenkins users rely on groovy init scripts to customize jenkins and enforce desired state. Those
-scripts directly invoke Jenkins API and as such can do everything (at your own risks). But they also require
+Experienced Jenkins users rely on groovy init scripts to customize jenkins and enforce desired state. Those
+scripts directly invoke Jenkins API and as such can do everything (at your own risk). But they also require
 you know Jenkins internals, and are confident in writing groovy scripts on top of Jenkins API.
 
 Configuration-as-Code plugin has been designed as an _**opinionated**_ way to configure jenkins based on 
 human-readable declarative configuration files. Writing such a file should be feasible without being a Jenkins
-expert, just translating into _code_ a configuration process one is used to execute in the web UI.
+expert, just translating into _code_ a configuration process one is used to executing in the web UI.
```
Comment (daniel-beck) [inline README.md]: This entire sentence makes no sense to me.

File: README.md [hunk 3]
```diff
@@ -54,7 +54,7 @@ There's no release yet.
 
 ## Examples
 
-Configuration file do include root entries for various components of your jenkins master installation. the `jenkins`
+Configuration file includes root entries for various components of your jenkins master installation. the `jenkins`
```
Comment (daniel-beck) [inline README.md]: *This configuration file includes* perhaps?

File: README.md [hunk 4]
```diff
@@ -100,7 +100,7 @@ Also see [demos](demos) folder with various samples.
 ## Full documentation
 
 The configuration file format depends on the version of jenkins-core and installed plugins. 
-Documentation is generated from a live instance, as well as JSON-schema you can use to validate configuration file
+Documentation is generated from a live instance, as well as a JSON-schema you can use to validate configuration file
```
Comment (daniel-beck) [inline README.md]: *validate configuration file*

**Comment by ndeloof** [comment]
merged

**Pull request closed without merging**